### PR TITLE
Q&A additional functionalities

### PR DESCRIPTION
**Pull request by tishvo**

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -181,13 +181,92 @@
 }
 
 .qaList {
-  height: 200px;
+  height: 250px;
   overflow: scroll;
-  border-style: solid;
 }
 
+<<<<<<< HEAD
```
Comment (anteofabris) [inline public/styles.css]: I'm not sure if this is part of the current markup on github or if this is still in the file... If we could double check before I merge tomorrow that would be sweet. Referring to the '<<<<< HEAD' writing on line 188.

File: public/styles.css [hunk 2]
```diff
@@ -181,13 +181,92 @@
...
+  display: none;
+}
+
+.tv_anc {
+  font-weight: normal;
+}
+
+.tv_underline {
+  text-decoration: underline;
+=======
```
Comment (anteofabris) [inline public/styles.css]: Same here.

File: public/styles.css [hunk 3]
```diff
@@ -181,13 +181,92 @@
...
 #app {
   width: 80%;
   margin: auto;
   max-width: 960px;
+>>>>>>> main
```
Comment (anteofabris) [inline public/styles.css]: And here. lol

**Review by anteofabris — COMMENTED** [review]
Besides the three comments, everything looks great! Your css code is really cool, especially the .qModal styling.

**Comment by tishvo** [comment]
great catches! (and TY for the review!) I went ahead and updated those lines in styles.css and committed

**Pull request merged**